compose/cli/errors.py

```diff
@@ -58,7 +58,7 @@ def __init__(self, url):
 class ComposeFileNotFound(UserError):
     def __init__(self, supported_filenames):
         super(ComposeFileNotFound, self).__init__("""
-        Can't find a suitable configuration file. Are you in the right directory?
+        Can't find a suitable configuration file. Checked every parent of the current directory. Are you in the right directory?
```

Comment:
How about: "Can't find a suitable configuration file in this directory or any parent. Are you in the right directory?"